README.md

```diff
@@ -44,16 +45,16 @@ The following flags can be used for each build:
 
 Flag | Description
 ---- | -----------
-`-tag` | The Vim release.  It must match the tags on Vim's [releases page](https://github.com/vim/vim/releases).
-`-name` | The name to use for the binary's symlink.  If omitted, the name will default to `vim-$TAG`.
+`-vim` | The Vim variant.  Either `vim` (default) or `nvim`.
```

Comment:
Yes, the name `-vim` is weird after all.
There might be other variants in the future though, so `-nvim 1` might not be sufficient..?!

But I am +0 on changing it like that.